qiita_db/data.py

```diff
@@ -229,19 +222,19 @@ def _set_link_filepaths_status(self, status):
         ValueError
             If the status is unknown
         """
-        self._check_subclass()
-        if (status not in ('idle', 'linking', 'unlinking') and
-                not status.startswith('failed')):
-            msg = 'Unknown status: %s' % status
-            LogEntry.create('Runtime', msg,
-                            info={self.__class__.__name__: self.id})
-            raise ValueError(msg)
-
-        conn_handler = SQLConnectionHandler()
-        conn_handler.execute(
-            "UPDATE qiita.{0} SET link_filepaths_status = %s "
-            "WHERE {1} = %s".format(self._table, self._data_filepath_column),
-            (status, self._id))
+        with TRN:
+            if (status not in ('idle', 'linking', 'unlinking') and
+                    not status.startswith('failed')):
```

Comment:
flake8 treats if statements differently for indentation, since the things in the if statement are already indented once. This means the next line of the if needs to be indented twice so you can tell the difference between lines in the if and the continuation of the if.